app/models/event.rb

```diff
@@ -18,4 +19,17 @@ def data
       status: status
     }
   end
+
+  private
+
+  def date_validation
+    return unless !self[:finish_date].nil? && !self[:start_date].nil?
+
+    if self[:finish_date] < self[:start_date]
+      errors.add(:finish_date, "can't be less than start date")
```

Comment:
Use localization for error text: https://stackoverflow.com/questions/17935597/ruby-on-rails-i18n-want-to-translate-custom-messages-in-models